source/draw/Color.ooc

```diff
@@ -135,7 +136,7 @@ ColorRgb: cover {
 		This new(Float mix(a r, b r, ratio) as Byte, Float mix(a g, b g, ratio) as Byte, Float mix(a b, b b, ratio) as Byte)
 	}
 	distance: func (other: This) -> Float {
-		((this r - other r) as Float pow(2) + (this g - other g) as Float pow(2) + (this b - other b) as Float pow(2)) / 3.0f sqrt()
+		((this r - other r) as Float pow(2) + (this g - other g) as Float pow(2) + (this b - other b) as Float pow(2)) sqrt()
```

Comment:
`distance(a, b)` should equal `distance(a toRgba(), b toRgba())` because it is still the same perceived colors. The problem comes when one adds a uniform channel and wants to have the same distance.